### PR TITLE
refactor(Quickstarts): Updated the meta details of 26 quickstarts

### PR DESCRIPTION
**Pull request by smamindla57**
update the page title and meta description of top 26 quick-starts.
list of the quickstarts
1. Amazon web services
2. Apache
3. CircleCI
4. CodeStream
5. Databricks
6. Django
7. .Net
8. Express
9. FastAPI
10. Flask
11. IBM MQ
12. Java
13. Kafka
14. Lacework
15. Laravel
16. MongoDB
17. MySQL
18. NGINX
19. PHP
20. Redis
21. Salesforce Event logs
22. Slack notifications
23. Synthetics SSL certification
24. Apache Tomcat
25. WordPress
26. Python

### CODE REVIEW

File: src/data/quickstart-metadata.json [hunk 1]
```diff
@@ -2,5 +2,109 @@
...
+        "title": "MongoDB Performance Monitoring | New Relic",
+        "description": "Learn how MongoDB monitoring from New Relic helps business users and developers monitor performance. Get started today."
+    },
+    "mysql": {
+        "title": "MySQl Database Performance Monitoring | New Relic",
+        "description": "Learn how MySQL database monitoring from New Relic helps business users and developers monitor performance. Get started today."
+    },
+    "nginx": {
+        "title": "Nginx Performance Monitoring | New Relic",
+        "description": "Nginx web server monitoring tool by New Relic helps business users and developers monitor Kafka performance with New Relic. Install Now!"
```
Comment (mickeyryan42) [inline src/data/quickstart-metadata.json:80]: is this meant to say "monitor Nginx performance with New Relic" rather than Kafka?

**Review by zstix — COMMENTED** [review]
From a code perspective, this looks good. We'll want to have @seemantk review this for content.

**Review by seemantk — COMMENTED** [review]
These are all similar to each other, I'm not sure what extra value they provide.  I'd like to have the Marketing folks review these.

Please don't merge at this time.

**Review by aswanson-nr — APPROVED** [review]

**Review by zstix — APPROVED** [review]

**Comment by aswanson-nr** [comment]
Hey @smamindla57, this repository does not support pull request from forks. In the future could you submit pull requests from a branch on this repository? If you need access we can get you added!

**Comment by smamindla57** [comment]
Hi @aswanson-nr, yes please, grant me access. thank you

**Review by aswanson-nr — APPROVED** [review]

**Comment by aswanson-nr** [comment]
I built and tested this locally, looks good. The workflow won't run since this is from a fork so I'm going to force the merge, in the future lets keep PRs from branches within this repository instead of a fork workflow.

**Pull request merged**